Cyberwatch/cyberwatch-detection/_meta/fields.yml

```diff
@@ -0,0 +1,69 @@
+cyberwatch.active:
```

Comment:
```suggestion
cyberwatch.vas.active:
```
Based on `cyberwatch.vas.fixed_at`. What do you think @squioc ?

Reply:
Yes. I agree with you. I think all custom fields should be prefixed with `cyberwatch.vas`.